src/zui/components/ZUIAlert/index.tsx

```diff
@@ -0,0 +1,237 @@
+      return (
+        <CheckCircleOutline
+          sx={(theme) => ({
+            color: theme.palette.success.main,
+            fontSize: '1.25rem',
+          })}
+        />
+      );
+    } else {
+      //Severity must be "warning"
+      return (
+        <WarningAmberOutlined
+          sx={(theme) => ({
+            color: theme.palette.warning.dark,
+            fontSize: '1.25rem',
+          })}
+        />
+      );
+    }
+  };
+
+  const [animatedOut, setAnimatedOut] = useState(false);
+
+  const timeout = 1000;
+
+  useEffect(() => {
+    if (open) {
+      setAnimatedOut(false);
+    } else {
+      const timer = setTimeout(() => {
+        setAnimatedOut(true);
+      }, timeout);
+
+      return () => clearTimeout(timer);
+    }
+  }, [open]);
+
+  if (animatedOut) {
+    return null;
+  }
+
+  return (
+    <Fade appear={appear} in={open} timeout={timeout}>
+      <Box
+        sx={() => ({
+          backgroundColor: backgroundColors[severity],
+          borderRadius: '0.25rem',
+          color: colors[severity],
+          display: 'flex',
+          flexDirection: 'column',
+          padding: '1rem',
+        })}
+      >
+        <Box
+          sx={{
+            alignItems: 'flex-start',
+            display: 'flex',
+            gap: '0.75rem',
+            justifyContent: 'space-between',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              gap: '0.75rem',
+              paddingTop: onClose ? '0.25rem' : '',
+            }}
+          >
+            <Box
+              sx={{
+                display: 'flex',
+                paddingTop: '0.1rem',
+              }}
+            >
+              <Icon />
+            </Box>
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'column',
+                gap: '0.5rem',
+              }}
+            >
+              <ZUIText color="inherit" variant="bodyMdSemiBold">
+                {title}
+              </ZUIText>
+              {description && (
+                <ZUIText color="inherit" variant="bodySmSemiBold">
+                  {description}
+                </ZUIText>
+              )}
+            </Box>
+          </Box>
+          {onClose && (
+            <Button
+              color="inherit"
+              onClick={onClose}
+              sx={{
+                boxShadow: 'none',
+                flexShrink: 0,
+                minWidth: 0,
+                padding: '0.313rem',
+              }}
+            >
+              <Close sx={{ fontSize: '1.25rem' }} />
+            </Button>
+          )}
+        </Box>
+        {buttonProps && (
+          <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
+            <Button
+              color="inherit"
+              sx={(theme) => ({
+                boxShadow: 'none',
+                minWidth: 'max-content',
+                padding: '0.33rem 0.625rem',
```

Comment:
These numbers were whispered to us by an ancient UI rem monk 🧘‍♂️📐

Reply:
😂